code/game/objects/structures/flora.dm

```diff
@@ -185,13 +185,21 @@ GLOBAL_LIST_EMPTY(flora_uprooting_tools_typepaths)
 	. = FALSE
 	if(harvested || !harvestable)
 		return null
-	
+
 	if(harvesting_item)
-		if(!is_type_in_typecache(harvesting_item, required_tools))
-			return
+		//Check if its disallowed first, because we wanna cut it down in its tracks if so
 		if(is_type_in_typecache(harvesting_item, disallowed_tools))
 			return
```

Comment:
can you return FALSE instead since we're also using return TRUE, for consistency